ppafm/GUIWidgets.py

```diff
@@ -588,10 +583,8 @@ def updateView(self):
         data = self.FE[..., ic].transpose(2, 1, 0)
         self.figCan.plotSlice(data, iz, title=f"z = {z:.2f}Å")
 
-        if self.verbose > 0:
-            print("FFViewer.updateView", ic, iz, data.shape)
-        if self.verbose > 1:
-            print("updateView time [s]", time.perf_counter() - t0)
+        logger.debug(f"FFViewer.updateView {ic} {iz} {data.shape}")
+        perf_logger.info(f"updateView time [s] {time.perf_counter() - t0}")
```

Comment:
What is the `perf_logger` here? is it about performance - and that is instead of `verbose > 1` ?